### PR TITLE
Post review oskar

### PR DESCRIPTION
**Pull request by oskber**
Me and Max worked together on this using live share. 
I have added som conditionals to check if author and rating is added before sending post request.

### CODE REVIEW

**Review by AlbinNils98 — APPROVED** [review]
Nice job, looks good!

**Review by MadeleineDahlqvist — APPROVED** [review]
Nice, good job!

File: utils/reviewsUtils.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export function paginateSizeFive(page, arr) {
   return paginatedArr;
 }
 //Function that filters out reviews that aren't verified
+
```
Comment (richardolsson) [inline utils/reviewsUtils.js:37]: Avoid adding blank lines unnecessarily, as it makes reviewing more difficult when there are lots of little changes that don't actually mean anything. I recommend always reviewing my diffs before committing, either in VSCode, using `git diff` or using `git add -p` to review while staging (adding) for commit.

File: src/cmsAdapt.js [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
-      body: JSON.stringify(review)
-    })
-    console.log(res)
-    return res.json();
-  },
+
+    const author = review.data.author;
+    const rating = review.data.rating;
+
+       if (author && rating) {
```
Comment (richardolsson) [inline src/cmsAdapt.js:18]: The indentation of this line is off. I would expect it to be aligned with the `const`s above.

**Review by richardolsson — COMMENTED** [review]
I've added some feedback below.

**Pull request merged**